webextension/popup.js

```diff
@@ -101,6 +101,31 @@ function getShortCode(languageCode) {
     return languageCode.replace(/-.*/, "");
 }
 
+function suggestionClass(match) {
+    if(isSpellingError(match)){
+        return 'hiddenSpellError';
+    } else if (isSuggestion(match)){
+        return 'hiddenSuggestion';
+    } else {
+        return 'hiddenGrammarError';
+    }
+}
+
+function isSpellingError(match) {
```

Comment:
as there is already a variable called `isSpellingError` in the main logic area , `matchIsSpellingError` would be more appropriate to avoid confusion.  And then `isSuggestion` should become `matchIsSuggestion` to be consistent.